odh/base/monitoring/github-receiver.yaml

```diff
@@ -0,0 +1,33 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: github-receiver
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      run: github-receiver
+  template:
+    metadata:
+      labels:
+        run: github-receiver
+      annotations:
+        prometheus.io/scrape: 'true'
+        prometheus.io/port: '9990'
+    spec:
+      containers:
+        - name: github-receiver
+          image: quay.io/4n4nd/alertmanager-github-receiver:v0.10
```

Comment:
can we have this added to the operate-first quay repo [here](https://quay.io/organization/operate-first?tab=repos)?

Reply:
yes will do :+1: